Android.mk

```diff
@@ -20,6 +20,7 @@ LOCAL_PACKAGE_NAME := Mms
 LOCAL_JAVA_LIBRARIES += telephony-common mms-common
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit